libs/common/byteutils.hpp

```diff
@@ -44,6 +41,58 @@ namespace iroha {
     return array;
   }
 
+  /**
+   * Convert string of raw bytes to printable hex string
+   * @param str
+   * @return
+   */
+  inline std::string bytestringToHexstring(const std::string &str) {
+    std::stringstream ss;
+    ss << std::hex << std::setfill('0');
+    for (const auto &c : str) {
+      ss << std::setw(2) << (static_cast<int>(c) & 0xff);
+    }
+    return ss.str();
+  }
+
+  /**
+   * Convert printable hex string to string of raw bytes
+   * @param str
+   * @return
+   */
+  inline nonstd::optional<std::string> hexstringToBytestring(
+      const std::string &str) {
+    if (str.empty() or str.size() % 2 != 0) {
+      return nonstd::nullopt;
+    }
+    std::string result(str.size() / 2, 0);
+    for (size_t i = 0; i < result.length(); ++i) {
+      std::string byte = str.substr(i * 2, 2);
+      try {
+        result.at(i) = std::stoul(byte, nullptr, 16);
+      } catch (const std::invalid_argument &e) {
+        return nonstd::nullopt;
+      } catch (const std::out_of_range &e) {
+        return nonstd::nullopt;
+      }
+    }
+    return result;
+  }
+
+  /**
+   * Generates new random string from lower-case letters
+   * @param len - size of string to generate
+   * @return generated string
+   */
+  inline std::string randomString(int len) {
```

Comment:
Shouldn't be here

Reply:
Well, I think there are no better place in a whole project and I do not want to create one more file for just one stupid function.

Reply:
Literally, all think like you and this idea bring a chaos to project. =(